components/dialog/dialog.js

```diff
@@ -56,6 +57,14 @@ class Dialog extends DialogMixin(LitElement) {
 		`];
 	}
 
+	static get resources() {
+		return {
+			'en': {
+				close: 'Close this dialog'
```

Comment:
We probably have this lang term translated already in the LMS? Could probably snag all the translations...

Reply:
I'll take a look!